From 94256ecbcb3390afa8737667d2bb3f82f5f5c2fe Mon Sep 17 00:00:00 2001
From: Marek Belisko <marek.belisko@streamunlimited.com>
Date: Fri, 11 Apr 2014 14:30:26 +0200
Subject: [PATCH] config:stream800: Undef CONFIG_SPL_OS_BOOT

This avoid printing in SPL:
SPL: Please implement spl_start_uboot() for your board
SPL: Direct Linux boot not active!

Signed-off-by: Marek Belisko <marek.belisko@streamunlimited.com>
---
 include/configs/stream800.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/stream800.h b/include/configs/stream800.h
index a8c6967..d21febc 100644
--- a/include/configs/stream800.h
+++ b/include/configs/stream800.h
@@ -23,6 +23,8 @@
 
 #include <configs/ti_am335x_common.h>
 
+#undef CONFIG_SPL_OS_BOOT
+
 #define CONFIG_DMA_COHERENT
 #define CONFIG_DMA_COHERENT_SIZE	(1 << 20)
 
-- 
2.7.4

